Presentation.Web/app/components/global-admin/global-admin-system.view.html

```diff
@@ -6,6 +6,6 @@
 <div data-global-option-list="" dir-id="dataTypeId" title="Datatyper" state="global-admin.system.edit-system-types" data-options-url="odata/DataTypes" option-type="type"></div>
 <div data-global-option-list="" dir-id="frequencyTypeId" title="Frekvenser" state="global-admin.system.edit-system-types" data-options-url="odata/FrequencyTypes" option-type="type"></div>
 <div data-global-option-list="" dir-id="interfaceTypeId" title="Grænseflader" state="global-admin.system.edit-system-types" data-options-url="odata/InterfaceTypes" option-type="type"></div>
-<div data-global-option-list="" dir-id="sensitivePersonalDataTypeId" title="Følsomme persondata" state="global-admin.system.edit-system-types" data-options-url="odata/SensitivePersonalDataTypes" option-type="type"></div>
-<div data-global-option-list="" dir-id="registerTypeId" title="Kategorier af registrerede i databehandlingen" state="global-admin.system.edit-system-types" data-options-url="odata/RegisterTypes" option-type="type"></div>
+<div data-global-option-list="" dir-id="sensitivePersonalDataTypeId" title="Følsomme persondata" state="global-admin.system.edit-system-types" data-options-url="odata/SensitivePersonalDataTypes" disable-edit="true" option-type="type"></div>
```

Comment:
ahh sorry... for global admin, only the description box should be removed so in this case it is not ok to disable the entire edit functionality... so..

- For global admin:
- disable-description
- -- Remove from dialog and remove column in the grid

Reply:
@a-naskret ☝️